hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/constants/AbfsHttpConstants.java

```diff
@@ -149,7 +150,7 @@ public final class AbfsHttpConstants {
   public enum ApiVersion {
 
     DEC_12_2019("2019-12-12"),
-    APR_10_2021("2021-04-10"),
+    AUG_06_2021("2021-08-06"),
```

Comment:
No the encryptionContext header is supported only after this date for CPK. Since this version is used for CPK explicitly, better to use where all the headers are available ?

Reply:
CPK support was there on trunk already, so version would have been set. Is it the case CPK has different version on DFS and Blob endpoint??
Or in trunk the version was set wrong?

Reply:
Let's make changes here as per the discussions offline.